ios-xcframework/Config/Gutenberg-Shared.xcconfig

```diff
@@ -3,6 +3,12 @@ CLANG_ANALYZER_NONNULL = YES
 CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE
 CLANG_CXX_LANGUAGE_STANDARD = gnu++20
 CLANG_ENABLE_MODULES = YES
+// If CLANG_ENABLE_MODULE_DEBUGGING is YES (or not specified, which defaults to YES), then CocoaPods says:
+//
+// Linking a static library that was built with -gmodules, but the module cache was not found.
+// Redistributable static libraries should never be built with module debugging enabled.
+// The debug experience will be degraded due to incomplete debug information.
+CLANG_ENABLE_MODULE_DEBUGGING = NO;
```

Comment:
This change was [part of the previous PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5740). For it to be here, I must have messed up a `git rebase` resulting in that commit not landing in `trunk`.